pkgs/applications/video/kodi/default.nix

```diff
@@ -177,7 +177,7 @@ in stdenv.mkDerivation {
       libogg libvorbis flac libxslt systemd
       lzo libcdio libmodplug libass libbluray
       sqlite libmysqlclient avahi lame
-      curl bzip2 zip unzip glxinfo
+      curl bzip2 zip glxinfo
```

Comment:
Looks like this has not been removed from the argument list.

Reply:
on `staging` it is there and unused. On `staging-next it is also in `buildInputs`.